likelihood/exact_survivor.h

```diff
@@ -146,7 +147,7 @@ namespace DCProgs {
       //! Reference to eigenvalues
       t_rvector const & eigenvalues() const { return eigenvalues_; }
       //! A null matrix of appropriate size
-      decltype(t_rmatrix::Zero(1,1)) zero() const { return t_rmatrix::Zero(nopen, nopen); };
+      decltype(t_srmatrix::Zero(1,1)) zero() const { return t_srmatrix::Zero(nopen, nopen); };
```

Comment:
FYI, Zero is an expression, rather than an actual matrix. So changing this line probably has no effect.